src/Uno.UI.Tests.ViewLibraryProps/Uno.UI.Tests.ViewLibraryProps.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
 	<PropertyGroup>
 		<TargetFrameworks>net461</TargetFrameworks>
-		<NoWarn>1998;4014;67;CA1014</NoWarn>
+		<NoWarn>1998;4014;67</NoWarn>
```

Comment:
Nopefully, we no longer need this due to https://github.com/dotnet/roslyn-analyzers/pull/6008